update.sh

```diff
@@ -81,9 +79,10 @@ DeleteRunnerPrompt() {
     read -r -p "Do you really want to permanently delete this runner? <y/N> " prompt
     if [[ $prompt == "y" || $prompt == "Y" || $prompt == "yes" || $prompt == "Yes" ]]; then
       DeleteRunner
+      install_complete=true
```

Comment:
I'm not a fan of reusing this.

We should add a new variable to the `DeleteRunner()` function after the echo called `delete_complete` and update `RestartLutris` to use `&& [ "$install_complete" = true || "$delete_complete" = true ]; then`